src/components/selectYear.js

```diff
@@ -0,0 +1,111 @@
+import React, { PureComponent, PropTypes } from 'react'
+import { connect } from 'react-redux'
+import * as actions from '../actions/actionIndex'
+import reactCSS from 'reactcss'
+import store from '../store'
+
+const styles = reactCSS({
+  'default': {
+    button: {
+      color: 'orange',
+      fontSize: '16px',
+    }
+  }
+})
+
+const { object, func} = PropTypes
+
+class SelectYear extends PureComponent {
+
+  constructor() {
```

Comment:
I think we talked about this component during our session earlier today. So i'll wait to comment on this component until after your latest changes are committed/pushed.